packages/internal/src/utils/logger.ts

```diff
@@ -11,7 +11,17 @@ import {
   RULES_HELP,
 } from './logger.rules';
 
-const overrideRules: OverrideRule = require('./../ama.rules.json');
+const defaultRules: OverrideRule = require('./../ama.rules.json');
+let projectRules = defaultRules;
+try {
+  // look upwards to user's project root
+  // e.g we are here:
+  // root/node_modules/@react-native-ama/internal/dist/utils/logger.ts
+  const userDefinedRules: OverrideRule = require('./../../../../../ama.rules.json');
+  projectRules = Object.assign(projectRules, userDefinedRules);
+} catch (error) {
+  // noop
+}
```

Comment:
This is a much better approach as the user doesn't need to remember to copy their rules into the node modules.